client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -139,14 +144,19 @@ class DomainRegistrationSuggestion extends React.Component {
 			} );
 
 			buttonStyles = { ...buttonStyles, primary: false };
+		} else if ( isSignupStep ) {
+			buttonContent = translate( 'Select', { context: 'Domain mapping suggestion button' } );
```

Comment:
Could we just do `buttonContent  = buttonCopyTest` here?

Reply:
My thoughts were that in the context of signup it makes more sense to always be `Select`, since you won't be going immediately to checkout in that case. ([ref](https://github.com/Automattic/wp-calypso/issues/32526#issuecomment-485954546))

If you think `Purchase` also makes sense at that step in signup, we could just completely remove the `else if` here.

Reply:
No, that's a good point 👍